README.md

```diff
@@ -2,14 +2,29 @@
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
+  
+  1. The assumption of inputs normally being large: Asymptotic analysis typically assumes that input is large often approaching infinity. However, real world problems can typically involve finite input sizes, and some times those inputs are not large enough for asymptotic analysis to be fully realized. For instance, an algorithm that has a time complexity of O(n^2) may have similar performance when compared to O(n log n) when the input is small. Large inputs refering to values in the thousands or millions while small inputs refering to relatively small like 100 elements.
```

Comment:
So for anything less than input size 100, asymptotic analysis is misleading? That's just not true.